bin/karma

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+./node_modules/karma/bin/karma "$@"
```

Comment:
You're right. I didn't know about this and have documented these commands instead.